### PR TITLE
UI Sanding

### PR DESCRIPTION
**Pull request by drfarrell**
### Description

Improvements to...
- Node tree – making it more obvious what elements belong to a component (purple text)
- Project Settings page – better typography, glossy background, better close button, and better text labels
- Project sub-menu dropdown – make this go away after the user has selected an item from the menu
- Inputs – made the quad-icon not cut-off on the sides, tightened the spacing of the accordions 

### What is the purpose of this pull request? 

- [X] New feature
- [ ] Documentation update
- [X] Bug fix
- [ ] Refactor
- [ ] Release
- [ ] Other

### CODE REVIEW

**Comment by drfarrell** [comment]
I'm still going to work on this a bit – but just saving this here.

File: apps/studio/src/routes/editor/LayersPanel/Tree/TreeNode.tsx [hunk 1]
```diff
@@ -174,18 +193,20 @@ const TreeNode = observer(
...
                             )}
                         >
-                            <span className="w-4 h-4 flex-none">
+                            <span className="w-4 h-4 flex-none relative">
                                 {!node.isLeaf && (
                                     <div
-                                        className="w-4 h-4 flex items-center justify-center"
-                                        onClick={() => node.toggle()}
+                                        className="w-4 h-4 flex items-center justify-center absolute z-50"
+                                        onMouseDown={(e) => {
```
Comment (Kitenite) [inline apps/studio/src/routes/editor/LayersPanel/Tree/TreeNode.tsx:203]: What's the intention here?

Comment (Kitenite) [reply apps/studio/src/routes/editor/LayersPanel/Tree/TreeNode.tsx:203]: I guess select the node when it's toggled?

**Review by Kitenite — APPROVED** [review]
Looks good, feel free to merge it if the one question is intentional

**Review by Kitenite — CHANGES_REQUESTED** [review]
Actually, found a visual bug from the animation

https://github.com/user-attachments/assets/10011228-c524-4754-9320-1785a20b5624

File: apps/studio/src/routes/editor/LayersPanel/Tree/TreeNode.tsx [hunk 2]
```diff
@@ -138,6 +138,16 @@ const TreeNode = observer(
             node.data.isVisible = !node.data.isVisible;
         }
 
+        function hasComponentAncestor(node: NodeApi<LayerNode>): boolean {
```
Comment (Kitenite) [inline apps/studio/src/routes/editor/LayersPanel/Tree/TreeNode.tsx:141]: This is a very expensive operation. With a large enough tree, the app will grind to a halt.

Comment (Kitenite) [reply apps/studio/src/routes/editor/LayersPanel/Tree/TreeNode.tsx:147]: Fixed! Good thing too because I added some extra performance fix thanks to this.

**Review by Kitenite — APPROVED** [review]

**Pull request merged**